### ISSUE TITLE
Try reducing the learning rate again at some point

### ISSUE THREAD

**Issue by soi** [weberlab-hhu/Helixer#32]
Maybe it now works with DanQ

**Comment by soi** [comment]
https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/schedules/ExponentialDecay
